GitUI/CommandsDialogs/BrowseDialog/DashboardControl/DashboardCategory.cs

```diff
@@ -71,8 +71,8 @@ public void DisableContextMenu()
 
         private void SetUpFonts()
         {
-            _NO_TRANSLATE_Caption.Font = new Font(SystemFonts.MessageBoxFont.FontFamily, 10, FontStyle.Bold,
-                                                  GraphicsUnit.Point);
+            _NO_TRANSLATE_Caption.Font = FontUtil.MainInstructionFont;
+            _NO_TRANSLATE_Caption.ForeColor = FontUtil.MainInstructionColor;
```

Comment:
I'm aware this will be overwritten when #3693 is merged.

Reply:
I'll try to get that one in sooner than later, so you could help in making it prettier :)